core/embed/prodtest/towncrier.toml

```diff
@@ -0,0 +1 @@
+../../../tools/towncrier.toml
```

Comment:
Dead link?

Reply:
it's not dead locally :woman_shrugging: i think this is github bug. `core/ ( embed/ ( prodtest / .. ) /.. ) /.. / tools/towncrier.toml` exists